mu/app.py

```diff
@@ -77,9 +77,7 @@ def __init__(self, animation, parent=None):
         self.animation = animation
         self.animation.frameChanged.connect(self.set_frame)
         # Always on top.
-        super().__init__(
-            self.animation.currentPixmap(), Qt.WindowStaysOnTopHint
-        )
+        super().__init__(self.animation.currentPixmap(), Qt.WindowStaysOnTopHint)
```

Comment:
This line looks longer than 79 characters.

I think it's possible the issue you are having with black is that maybe you are running the `black` command directly?
If so make sure you use the Makefile/make.py `make tidy` to ensure the specific rules for the project are applied.